src/Core/Framework/Api/ApiDefinition/Generator/Schema/AdminApi/paths/consent.json

```
@@ -0,0 +1,109 @@
+{
+  "openapi": "3.1.0",
+  "info": [],
+  "paths": {
+    "\/consents": {
+      "get": {
+        "tags": [
+          "Consent Management"
+        ],
+        "summary": "List all consents for current user",
+        "description": "Returns a list of all available consents with their current status for the authenticated admin user. Each consent includes its name, identifier, and current state (requested, accepted, or revoked).",
+        "operationId": "fetchConsents",
+        "responses": {
+          "200": {
+            "description": "List of consents with their status",
+            "content": {
+              "application\/json": {
+                "schema": {
+                  "$ref": "#\/components\/schemas\/consentListResponse"
+                },
+                "example": [
+                  {
+                    "name": "data-collection",
+                    "identifier": "user-123",
+                    "status": "accepted",
+                    "actorId": "user-123"
+                  },
+                  {
+                    "name": "analytics-tracking",
+                    "identifier": "user-123",
+                    "status": "requested",
+                    "actorId": "user-123"
+                  }
+                ]
```

examples are missing scope, proposition based on what I understood of what fields are possible
```suggestion
                "example": [
                  {
                    "name": "data-collection",
                    "scope": "global",
                    "identifier": "user-123",
                    "status": "accepted",
                    "actorId": "user-123"
                  },
                  {
                    "name": "analytics-tracking",
                    "scope": "admin_user",
                    "status": "requested",
                    "actorId": "user-123"
                  }
                ]
```

---

I think this comes from the former approach, when we didn't send the scope at all. I'm not sure if the users need that information or if they should only get a list of consents they are affected by